pkgs/tools/misc/blacken-docs/default.nix

```diff
@@ -0,0 +1,38 @@
+{ black
+, fetchFromGitHub
+, lib
+, python3
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "blacken-docs";
+  version = "1.15.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "adamchainz";
+    repo = "blacken-docs";
+    rev = version;
+    hash = "sha256-3FGuFOAHCcybPwujWlh58NWtuF5CebaKTgBWgCGpSL8=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+  ];
+
+  propagatedBuildInputs = [
+    black
+  ];
+
+  nativeCheckInputs = [
+    black
+    python3.pkgs.pytestCheckHook
+  ];
+
+  meta = {
+    homepage = "https://github.com/adamchainz/blacken-docs";
+    changelog = "https://github.com/adamchainz/blacken-docs/blob/${src.rev}/CHANGELOG.rst";
+    description = " Run Black on Python code blocks in documentation files";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.l0b0 ];
```

Comment:
`with maintainers ; [ l0b0 ];`

I remember seeing discussion about this, I couldn't find it, maybe some one can link. While I am actually fine with this line as it.  suggested way seems to be preferred in nixpkgs